app/views/guests/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Guest Show Page</h1>
+<section id="guest_info">
+<p><%= @guest.name %></p>
+<p><%= @guest.nights %></p>
+<% @guest.rooms.each do |room| %>
+    <%= room.suite %>
+    <br>
+    <%= room.rate %>
+    <br>
+    <%= room.hotel.name %>
+    <br>
+<% end %>
+</section>
+<section id="room_form">
+<%= form_with url: "/guest/#{@guest.id}", method: :post, local: true do |form| %>
+    <br>
+    <%= form.label :room_id %>
+    <%= form.text_field :room_id %>
+    <%= form.submit "Add Room" %>
+<% end %>
```

Comment:
Great work! Your route on line 15 could be a bit more restful since you're actually creating a GuestRoom. It could go to something like '/guest_rooms' and pass the guest id via a hidden field in the form. Or, it might even make more sense to have something like "/guests/:id/rooms"